### PR TITLE
Label positioning in FindReplaceDlg + layout improvement

### LINKED ISSUE #11318: Labels in Find dialog moved unintentionally in v8.2.1+
### Description of the Issue
After Notepad++ 8.2 the labels in the Find dialog are a offset making it overflow in some non-Englisch translations.

Here's how it looked in 8.2:
2022-03-02 09_20_10-Notepad++8 2

And here's how it looks right now:
2022-03-02 09_20_10-Notepad++8 2 1_and_above

Please note the "Find what", "Replace with", "Filters" texts.

### Steps to Reproduce the Issue
1. Open the Find dialog using CTRL+F

### Expected Behavior
Positioning as in Notepad++ 8.2

### Actual Behavior
Positioning is a bit offset.

### Debug Information
Notepad++ v8.3.2   (64-bit)
Build time : Feb 27 2022 - 03:35:46
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Pro (64-bit) 
OS Version : 2009
OS Build : 19044.1526
Current ANSI codepage : 1252
Plugins : ComparePlugin.dll JSMinNPP.dll mimeTools.dll NppConverter.dll NppExport.dll PythonScript.dll XMLTools.dll

### PR DESCRIPTION
**Pull request by dfs-**
* move labels to resemble v8.2 positioning
* resize IDD_FINDINFILES_DIR_COMBO to make it end with the other comboboxes
* resize IDD_FINDINFILES_BROWSE_BUTTON to match combobox height
* make IDD_FINDREPLACE_SWAP_BUTTON in line with IDD_FINDINFILES_BROWSE_BUTTON

Fixes #11318 and improves the layout

**New Layout:**
pr

### CODE REVIEW

**Comment by donho** [comment]
The result of this PR to me is not nice:

image

**Comment by dfs-** [comment]
@donho 

It‘s on Windows 11 I guess? I‘ll take a look at it.

**Comment by donho** [comment]
@dfs- 
Yes, it's Windows 11

**Comment by dfs-** [comment]
@donho 
Tested the dialog on a newly installed Windows 11 VM and got the following results:

(Windows Scaling: 100%)
win11findreplacedlg_scale100


(Windows Scaling: 125%)
win11findreplacedlg_scale125

Is there anything I am missing? Do you have a setting enabled/disabled, that is not the default. I always test implementations without any additional configurations...

**Comment by rddim** [comment]
Is it possible to center in horizontal position the switching button and the three dots? Some pixels to the right will be good. Also from the screenshots, windows scaling 125% cut the `newline` text.

**Comment by dfs-** [comment]
> Is it possible to center in horizontal position the switching button and the three dots? Some pixels to the right will be good.
You mean moving them in the center between the Input fields and the other buttons? Definitely possible but might be irritating as to which part of the dialog they refer. 

> Also from the screenshots, windows scaling 125% cut the `newline` text.
I'll expand the GroupBox a little.

**Comment by rddim** [comment]
> moving them in the center between the Input fields and the other buttons? 
>
Yes, exactly

> to which part of the dialog they **refer**.
>
You are right, I didn't think from this position. So the 1st ask is not needed :)

**Comment by dfs-** [comment]
Hi @donho,

I tested the new layout with 150% scaling and can not reproduce your outcome.
image

Any advice?

**Comment by donho** [comment]
@dfs- 
225%

image

**Comment by donho** [comment]
@dfs- 

Just did the following modifications:

```diff
-    RTEXT           "&Find what :",IDFINDWHAT_STATIC,1,22,73,8
+    RTEXT           "&Find what :",IDFINDWHAT_STATIC,1,21,73,8^M
     COMBOBOX        IDFINDWHAT,76,20,170,150,CBS_DROPDOWN | CBS_AUTOHSCROLL | WS_TABSTOP
-    RTEXT           "Rep&lace with :",ID_STATICTEXT_REPLACE,1,40,73,8
+    RTEXT           "Rep&lace with :",ID_STATICTEXT_REPLACE,1,39,73,8^M
     COMBOBOX        IDREPLACEWITH,76,38,170,50,CBS_DROPDOWN | CBS_AUTOHSCROLL | WS_TABSTOP
     PUSHBUTTON      "",IDD_FINDREPLACE_SWAP_BUTTON,248,27,16,17
-    RTEXT           "Filter&s :",IDD_FINDINFILES_FILTERS_STATIC,1,58,73,8, SS_NOTIFY
+    RTEXT           "Filter&s :",IDD_FINDINFILES_FILTERS_STATIC,1,57,73,8, SS_NOTIFY^M
     COMBOBOX        IDD_FINDINFILES_FILTERS_COMBO,76,56,170,150,CBS_DROPDOWN | CBS_AUTOHSCROLL | WS_TABSTOP
-    RTEXT           "Dir&ectory :",IDD_FINDINFILES_DIR_STATIC,7,76,41,8
-    COMBOBOX        IDD_FINDINFILES_DIR_COMBO,50,74,196,150,CBS_DROPDOWN | CBS_AUTOHSCROLL | WS_TABSTOP
+    RTEXT           "Dir&ectory :",IDD_FINDINFILES_DIR_STATIC,1,75,73,8^M
+    COMBOBOX        IDD_FINDINFILES_DIR_COMBO, 76,74,170,150,CBS_DROPDOWN | CBS_AUTOHSCROLL | WS_TABSTOP^M

```
and got the result as following:

image

If the result of my modifications is acceptable in your scale - please integrate them and the PR will be accepted.

**Comment by dfs-** [comment]
@donho thanks for the fix, which I implemented as it didn’t seem to break anything.

**Comment by dfs-** [comment]
GCC doesn‘t like the modifier it seems. I‘ll only be able to have a look in a few hours. If anyone has a smart idea on how to resolve it, please chime in. :)

**Comment by dfs-** [comment]
I reverted the last two changes. It came to my attention that the problem is not the texts moving downward but rather the comboboxes not scaling in the same factor as other elements. One can notice the "..." button constantly getting bigger than the combobox left to it as scaling increases. Also the comboboxes increase their margin to another.

I am not sure if this is a bug in the operating system but [this Microsofts Dev Community entry](https://developercommunity.visualstudio.com/t/mfc-ownerdraw-combobox-doesnt-respect-dpi-scaling/107468) states it's not.

**Comment by dfs-** [comment]
@donho - GDI Scaling seems to do the trick (see comment before this one). I took a brief look in some of the dialogs possibly affected by changing the manifest and found no blatant issue.

**Comment by alankilborn** [comment]
>changing the manifest and found no blatant issue.

The last time the manifest was changed in order to fix problems, it broke more things that it fixed, and was reverted.

**Comment by dfs-** [comment]
> The last time the manifest was changed in order to fix problems, it broke more things than it fixed, and was reverted.

Quite of destroys my hopes for this whole PR. Nevertheless I found a practical improvement with GDI scaling enabled: starting Notepad++ on a different scaling screen will automatically resize the form to be about the same size as before. Without the GDI scaling it forced itself to have the same amount of pixels horizontally/vertically making it really small especially with my test setup for this PR (4K, 300% scaling)…

**Comment by Pyre909** [comment]
Just an fyi the font within the editor is blurry with Directwrite turned on.

Notepad++ v8.3.2   (64-bit)
Build time : Mar  5 2022 - 23:55:04
Path : C:\Users\rr200853\Desktop\npp\notepad++.exe
Admin mode : OFF
Local Conf mode : ON
Cloud Config : OFF
OS Name : Windows 10 Pro (64-bit) 
OS Version : 2009
OS Build : 19042.1526
Current ANSI codepage : 1252
Plugins : ComparePlugin.dll 

Capture

**Review by donho — CHANGES_REQUESTED** [review]
1. Please don't use ^M.
2. dpiAware.manifest should not be modified.

**Comment by donho** [comment]
@dfs- 
Could we close this PR if you have no intention to work on it?

**Comment by dfs-** [comment]
@donho 
Please bare with me as I am currently on vacation with no Visual Studio at hand. I‘ll be back on Monday. However you can close/reject this PR and I will create a new one.

File: PowerEditor/src/ScintillaComponent/FindReplaceDlg.rc [hunk 1]
```diff
@@ -24,16 +24,16 @@ EXSTYLE WS_EX_DLGMODALFRAME | WS_EX_WINDOWEDGE
 CAPTION "Replace"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
-    RTEXT           "&Find what :",IDFINDWHAT_STATIC,1,21,75,8
+    RTEXT           "&Find what :",IDFINDWHAT_STATIC,1,22,73,8^M
```
[the remark this answers is not in the document]
Comment (dfs-) [reply PowerEditor/src/ScintillaComponent/FindReplaceDlg.rc]: Don't quite understand the ^M part, GIT is configured to use core.autocrlf = true as the github doc suggests.

**Comment by donho** [comment]
@dfs- 
No rush.
Have good holidays then :)

**Comment by double-standard-you** [comment]
> The result of this PR to me is not nice:
> 
> image

American Shares Truth About Joshua Wong & Hong Kong Democracy
https://www.youtube.com/watch?v=8nXz66Btfl0

American Shares Truth on Xinjiang and Tibet Controversies in China
https://www.youtube.com/watch?v=CldtoYwPtMk

Why is Ukraine the West’s Fault? Featuring John Mearsheimer
https://www.youtube.com/watch?v=JrMiSQAGOS4

Bat coronavirus found in U.S.-funded bio-lab in Ukraine: Russian Defense Ministry
https://www.youtube.com/watch?v=BCydUeHAhzQ

Ukrainians were denied US visas as White House promise help to refugees
https://www.youtube.com/watch?v=BGZhzNFmyhc

Reports of racist treatment against Africans trying to flee Ukraine
https://www.youtube.com/watch?v=JItYOW_uT8s

Fleeing African and Indian students face racism at Ukraine border
https://www.youtube.com/watch?v=ODMOzwI__zs

Canadian RCMP use horses to trample protestors
https://www.youtube.com/watch?v=Usn3CR0E1v0

A bald-faced lie from the Western media corp: Critical “Uyghur Genocide” Questions: A CBC Interview Follow-up
https://www.youtube.com/watch?v=O7IbsPFM2Xg

**Comment by dfs-** [comment]
@donho 
I opened a new PR #11386 which should replace this one.
I am not closing this PR here right now, since I see a slight chance you dislike the new method and prefer this one.

**Comment by dfs-** [comment]
New PR was used.

**Pull request closed without merging**